### PR TITLE
Los2119/hw2

### PR DESCRIPTION
**Pull request by oforiwaasam**
Dear professor, 

I experienced a few challenges that made this part 2 especially challenging and I had to make some quick decisions:

The biggest issue was the fact that another student had a `matplotlib.md` file which raised additional merge conflicts. I wasn't sure about what to do ultimately so I just added my changes below theirs due to certain discrepancies. I'm not sure if that approach was the best but that left me with multiple attempts at part2 which ultimately rendered my pick and squash commands unexecuted, leaving multiple commits in this pr. I still have screenshots of all the work I did in my local repo before the deadline to show.

### CODE REVIEW

File: projects/converter.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# converter
```
Comment (timkpaine) [inline projects/converter.md:1]: remove this file, wrong folder

File: projects/matplotlib.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Matplotlib
```
Comment (timkpaine) [inline projects/matplotlib.md:1]: remove this file, wrong folder

File: students/los2119.md [hunk 3]
```diff
@@ -1,27 +1,14 @@
...
 ## Project Proposal
-
-For my project, I'm looking into a couple of ideas:
-
-- to build a browser extension to show how much time users are spending on specific sites
-- Another idea is to build a browser extension to keep track of shopping wishlists from shopping sites they've visited
-- Build an app to convert all sorts of units/currencies
-- Build an app to track food/grocery expiry dates and trial expiry dates
-- Build an app that would turn a user's phone silent based on their location and preference
+- [converter](./projects/python/converter.md)
```
Comment (timkpaine) [inline students/los2119.md:11]: update link path and re-squash

File: students/los2119.md [hunk 4]
```diff
@@ -1,27 +1,14 @@
...
 ## Project Contributions
-
-I'm planning on starting my open source contribution journey as an actual beginner taking baby steps so in terms of projects to contribute to, I would be checking out [GitHub's good first issue topics page](https://github.com/topics/good-first-issue) and the [First Timers Only](https://www.firsttimersonly.com/) tool for good first issues to contribute to.
-As I progress through my journey of open source, I would also like to make contributions to tools and libraries I use on a daily listed below:
-
-- Flask
-- Pandas
-- Matplotlib
-- React and React-native
+- [matplotlib](./projects/python/matplotlib.md)
```
Comment (timkpaine) [inline students/los2119.md:14]: update link path and re-squash

**Review by timkpaine — CHANGES_REQUESTED** [review]
make sure all commits squashed into 1

**Comment by oforiwaasam** [comment]
New pr with all resolved issues in a single commit [here](https://github.com/ColumbiaOSS/project-proposals-s2023/pull/370)

**Pull request closed without merging**